.github/workflows/lint.yaml

```diff
@@ -0,0 +1,22 @@
+name: Lint
```

Comment:
Is this file needed? If yes, maybe put a comment why, it doesn't seem related to the hooks.

Reply:
I used to have it in the commit message description: "add lint github actions check", but not it's not related to the hooks, perhaps won't be needed any more, so I will remove it.